examples/vacation_app/cmd/helper.go

```diff
@@ -12,6 +12,6 @@ func createBus(serviceName string) gbus.Bus {
 		WorkerNum(3, 1).
 		WithConfirms().
 		PurgeOnStartUp().
-		Txnl("mysql", "rhinof:rhinof@/rhinof").
+		Txnl("mysql", "rhinof:rhinof@/rhinof?parseTime=true").
```

Comment:
@kerenbe4 @danielwitz @vladshub this is an interesting side effect as it delegates the responsibility for grabbit to work correctly to the client, which I think is a bad idea.
I think grabbit should do the extra mile and identify if the parseTime=true is passed or not and adapt accordingly.
Meaning if parseTime=true then the saga store can scan the created_at column to a time.Time struct
if parseTime=false or is not passed in the saga store needs to scan the created_at to sql.RawBytes and parse the date out of it.

Reply:
Or even better scan the value into a mysql.NullTime and it will handle the parsing for you

Reply:
definitely agree with your first comment, I wanted to check that the parseTime=true will resolve the issue I had, so I just passed it through the connection string to get the build to pass before I continue in that path

about sql.NullTime, I tried it, but got an error that that type is not recognized.. maybe i missed something

Reply:
thee NullTime is defined in the mysql package not thee sql one.
So you will need to import "mysql" and use mysql.NullTime